 jasper 2.0.17 fixes some signed compare without breaking ABI
 but it breaks API
Index: DevIL/src-IL/src/il_jp2.cpp
===================================================================
--- a/DevIL/src-IL/src/il_jp2.cpp
+++ b/DevIL/src-IL/src/il_jp2.cpp
@@ -314,15 +314,15 @@ ILboolean iLoadJp2Internal(jas_stream_t
 
 
 
-static int iJp2_file_read(jas_stream_obj_t *obj, char *buf, int cnt)
+static int iJp2_file_read(jas_stream_obj_t *obj, char *buf, unsigned cnt)
 {
 	obj;
 	return iread(buf, 1, cnt);
 }
 
-static int iJp2_file_write(jas_stream_obj_t *obj, char *buf, int cnt)
+static int iJp2_file_write(jas_stream_obj_t *obj, const char *buf, unsigned cnt)
 {
 	obj;
 	return iwrite(buf, 1, cnt);
 }
 
